### DISCUSSION TITLE
.conf settings: empty vs default vs automatic - LOG message, in-line comments

### DISCUSSION THREAD

**Discussion by Torinde** [dosbox-staging/dosbox-staging#3966] [General]
1. I find it useful if as user I can keep a note about a setting as comment, e.g. `machine = cga ; cga for 1980s, svga_3d for 1990s`
2. LOG gives sometimes a yellow warning and other times a white message - I'm not certain what's the logic/choice for those situations:
  - if you run with "machine = " you get yellow warning in the LOG (and svga_3d as it's the default)
  - if you run with "language = " you get white LOG entry (not a warning) and automatic/default language

Is it that 'more important' settings get a warning if empty and cosmetic ones - just a message?

**Comment by johnnovak** [comment]
1. Put your comments on the preceding line, prefixed with `#`. In-line comments like that will never be supported, no way.
2. Warnings should always be yellow and errors are red. It's just for `language = ` INFO log level is used instead of WARN by mistake.
